webapp/channels/src/components/post_view/data_spillage_report/data_spillage_report.tsx

```diff
@@ -136,6 +138,34 @@ export function DataSpillageReport({post, isRHS}: Props) {
         };
     }, [formatMessage]);
 
+    const footer = useMemo(() => {
+        if (isRHS) {
+            return null;
+        }
+
+        return (<DataSpillageFooter post={post}/>);
+    }, [isRHS, post]);
+
+    const actionRow = useMemo(() => {
+        if (!reportedPost || !reportingUser) {
+            return null;
+        }
+
+        let showActionRow;
+        if (!propertyFields || !propertyValues) {
+            showActionRow = true;
+        } else {
+            const status = propertyValues.find((value) => value.field_id === propertyFields.status.id)?.value as string | undefined;
+            showActionRow = reportedPost && reportingUser && status && (status === 'Pending' || status === 'Assigned');
```

Comment:
nit: this strings `Pending` and `Assigned` should either be constants or an enum